config/images/images.yaml

```diff
@@ -21,7 +21,7 @@ images:
 - source: alpine
   destination: eu.gcr.io/gardener-project/3rd/alpine
   tags:
-  - 3.15.4
+  - 3.15.8
```

Comment:
We should keep the old version too, that we can keep track on the images we already copied for previous versions.

Reply:
If that's the decision we should state this explicitly in `config/images/README.md`

Reply:
I kept the old version, but I am also a bit surprised that we want to keep the history in addition to version control (git) also in the file.